packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1624,4 +1624,38 @@ describe('Picker', function () {
       expect(progressbar).not.toBeInTheDocument();
     });
   });
+
+  describe('disabled', function () {
```

Comment:
it looks like we have no tests about disabled, would you mind adding a few more asserts to verify that interactions don't trigger anything as well?